app/models/employee.rb

```diff
@@ -0,0 +1,2 @@
+class Employee < ApplicationRecord
```

Comment:
is this model related to the `Company` Model? if yes, add the relation here 
your can read about that here:
https://guides.rubyonrails.org/association_basics.html

Reply:
Done